build_scripts/build_linux_deb-2-installer.sh

```diff
@@ -108,12 +109,12 @@ if [ "$PLATFORM" = "arm64" ]; then
     --config ../../../build_scripts/electron-builder.json
   LAST_EXIT_CODE=$?
 else
-  echo electron-builder build --linux deb --x64 \
+  echo "${NPM_PATH}/electron-builder" build --linux deb --x64 \
     --config.extraMetadata.name=chia-blockchain \
     --config.productName="$PRODUCT_NAME" --config.linux.desktop.Name="Chia Blockchain" \
     --config.deb.packageName="chia-blockchain" \
     --config ../../../build_scripts/electron-builder.json
-  npx electron-builder build --linux deb --x64 \
```

Comment:
## For reviewers
`npx electron-builder` does not invoke `electorn-builder` installed in `build_scripts/npm_linux/node_modules because `build_script` dir is not direct parent of the current dir (`chia-blockchain-gui`)
Here I force to run `$NPM_PATH/electron-builder` in order to use `electron-builder` installed under `build_scripts/npm_linux`.